yetanotherco/aligned_layer
# fix(cli): Use ethers as `max_fee` unit in submit to aligned

# User ethers as unit in `max_fee` for submit to aligned

## Description

Use ethers as `max_fee` unit in submit to aligned.

### To Test:
- Run a local devnet.
- Successfully submit a proof with a `max_fee` denominated in `ethers`
```
cast send --rpc-url http://localhost:8545 0x7bc06c482DEAd17c0e297aFbC32f6e63d3846650 --value 1ether --private-key 0x47e179ec197488593b187f80a00eb0da91f1b9d0b13f8733639f19c30a34926a

cd batcher/aligned && cargo run --release -- submit \
                --proving_system SP1 \
                --proof ../../scripts/test_files/sp1/sp1_fibonacci.proof \
                --vm_program ../../scripts/test_files/sp1/sp1_fibonacci.elf \
                --repetitions 20 \
                --proof_generator_addr 0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65 \
                --rpc_url http://localhost:8545 \
                --network devnet --max_fee 0.01ether --private_key 0x47e179ec197488593b187f80a00eb0da91f1b9d0b13f8733639f19c30a34926a
```
- Your proof should be submitted to the batcher

## Type of change

Please delete options that are not relevant.


## Checklist

  - [Link to the open issue addressing it]()
